hls4ml/backends/fpga/fpga_types.py

```diff
@@ -71,6 +75,10 @@ def definition_cpp(self):
             self._saturation_mode_cpp(self.saturation_mode),
             self.saturation_bits,
         ]
+        if args[3] == 'AC_TRN' and args[4] == 'AC_WRAP' and args[5] == 0:
+            # This is the default, so we won't write the full definition for brevity
+            args[3] = args[4] = args[5] = None
```

Comment:
Does ac_fixed actually have saturation bits? It might be tangental, but I didn't see it in the documentation or a cursory look at the source code. If it's not supported, we should maybe add a warning if it's set. (I don't fully understand how those are used.)

Reply:
ok, I'll add the warning under this check to inform the user